src/utils/customErrorHandler.ts

```diff
@@ -7,7 +7,7 @@ export class CustomGraphQLError extends Error {
 
 export const formatError = (error:any) => {
     if (error.originalError instanceof CustomGraphQLError) {
-        return { message: error.message };
+        return { message: 'something went wrong',error:error.message };
```

Comment:
the value of error.message is still not going to be human understandable, what's your take on this?